### ISSUE TITLE
Snapshots appear to fail but new one eventually appears

### ISSUE THREAD

**Issue by genebean** [home-assistant/supervisor#2663]
### Describe the issue you are experiencing

When I take a full snapshot, it tells me it has failed and to see Supervisor. If I wait long enough, a new snap shot will still appear, though I do not know if it is restorable.

### What is the used version of the Supervisor?

supervisor-2021.02.11

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### What is the version of your installed operating system?

5.12

### What version of Home Assistant Core is installed?

core-2021.2.3

### Steps to reproduce the issue

1. take full snapshot
2. wait a while
3. view logs while waiting
4. look at list of snapshots

### Anything in the Supervisor logs that might be useful for us?

```txt
21-03-03 02:45:57 INFO (MainThread) [supervisor.snapshots] Found 4 snapshot files
21-03-03 02:46:05 INFO (MainThread) [supervisor.snapshots] Creating new full-snapshot with slug 984bcca7
21-03-03 02:46:05 INFO (MainThread) [supervisor.snapshots] Snapshotting 984bcca7 store Add-ons
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_airsonos
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_airsonos
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_samba
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_samba
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_ssh
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_ssh
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_check_config
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_check_config
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_zerotier
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_zerotier
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_grafana
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_grafana
21-03-03 02:46:05 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_influxdb
21-03-03 02:47:05 WARNING (MainThread) [supervisor.jobs] 'SnapshotManager.do_snapshot_full' blocked from execution, system is not running - CoreState.FREEZE
21-03-03 02:47:35 INFO (MainThread) [supervisor.snapshots] Found 4 snapshot files
21-03-03 02:47:49 INFO (MainThread) [supervisor.snapshots] Found 4 snapshot files
21-03-03 02:47:49 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_influxdb
21-03-03 02:47:49 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_configurator
21-03-03 02:47:49 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_configurator
21-03-03 02:47:49 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_esphome
21-03-03 02:51:32 INFO (MainThread) [supervisor.snapshots] Found 4 snapshot files
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_esphome
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_dnsmasq
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_dnsmasq
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_mosquitto
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_mosquitto
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_logviewer
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_logviewer
21-03-03 02:52:01 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_vscode
21-03-03 02:52:12 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_vscode
21-03-03 02:52:12 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_nodered
21-03-03 02:52:12 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_nodered
21-03-03 02:52:12 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_deconz
21-03-03 02:52:12 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_deconz
21-03-03 02:52:12 INFO (MainThread) [supervisor.snapshots] Snapshotting 984bcca7 store folders
21-03-03 02:52:12 INFO (SyncWorker_7) [supervisor.snapshots.snapshot] Snapshot folder media
21-03-03 02:54:47 INFO (SyncWorker_7) [supervisor.snapshots.snapshot] Snapshot folder media done
21-03-03 02:54:47 INFO (SyncWorker_6) [supervisor.snapshots.snapshot] Snapshot folder share
21-03-03 02:54:47 INFO (SyncWorker_6) [supervisor.snapshots.snapshot] Snapshot folder share done
21-03-03 02:54:47 INFO (SyncWorker_1) [supervisor.snapshots.snapshot] Snapshot folder addons/local
21-03-03 02:54:47 INFO (SyncWorker_1) [supervisor.snapshots.snapshot] Snapshot folder addons/local done
21-03-03 02:54:47 INFO (SyncWorker_2) [supervisor.snapshots.snapshot] Snapshot folder ssl
21-03-03 02:54:47 INFO (SyncWorker_2) [supervisor.snapshots.snapshot] Snapshot folder ssl done
21-03-03 02:54:47 INFO (SyncWorker_0) [supervisor.snapshots.snapshot] Snapshot folder homeassistant
21-03-03 02:55:15 INFO (MainThread) [supervisor.snapshots] Found 4 snapshot files
21-03-03 02:56:07 INFO (SyncWorker_0) [supervisor.snapshots.snapshot] Snapshot folder homeassistant done
21-03-03 02:56:15 INFO (MainThread) [supervisor.snapshots] Creating full-snapshot with slug 984bcca7 completed
```

**Comment by pvizeli** [comment]
Fixed on next release

**Comment by genebean** [comment]
Thanks @pvizeli

**Comment by pvizeli** [comment]
Sorry, solved with new Core 2021.3.0 and new Supervisor 2021.03.1 (Currently on the beta channel)
